R/formula.R

```diff
@@ -0,0 +1,64 @@
+#' Building formula for model
```

Comment:
```suggestion
#' Building Model Formula
```
(just a bit more title style)